### DISCUSSION TITLE
Windows Development to Linux host issues

### DISCUSSION THREAD

**Discussion by sawyer1776** [pocketbase/pocketbase#1634] [Q&A] [answered]
So I may have just made a big mistake in my development process. I have been developing my App with PB backend on windows and have not had any issues, but now I am trying to put it into production and a Linux server seems to be the best option when I try to upload the PB data to the folder on the Linux server I get this error:

 panic: Failed to apply migration 1673167670_multi_match_migrate.go: SQL logic error: no such column: options

Is there an obvious route between windows for dev and then Linux for deployment or do I need to rebuild or host on windows?

Thanks!

**Comment by ganigeorgiev** [comment, marked as answer]
Changing the binary platform shouldn't result in the above error. I think this is similar to https://github.com/pocketbase/pocketbase/issues/1579.

My guess is that locally you are using an older PocketBase version (< v0.8) but for the production server you've probably deployed a newer (v0.8+) version.

v0.8+ has breaking changes and requires some manual migration steps. If that's the case, please follow the migration guide from https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md#migrate-from-v07x 
(_the temporary `upgrade` command is still available in the latest v0.11.3 binary so you should be able to upgrade directly to it following the steps in the guide_).

**Reply by sawyer1776** [reply]
Thanks so much, this is the issue! I was using v 0.7.6  working on upgrading now, thanks again!
